src/proto/Solana.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package TW.Solana.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message Transfer {
+    bytes private_key = 1;
+    string recipient = 2;
+    uint64 value = 3;
+}
+
+message Stake {
+    bytes from_private_key = 1;
+    bytes stake_private_key = 2;
+    string vote_pubkey = 3;
```

Comment:
because we gonna pass `private_key`, can you remove `string vote_pubkey = 3` from protobuf and derive it inside C++ code instead? In this case interface would be cleaner to use, caller should not be aware of the public key type and etc

Reply:
The vote_pubkey identifies the validator to stake, so it can't be derived from any of the other inputs.

Reply:
Make sense, thanks for clarifying